doc/plugins.rst

```diff
@@ -116,7 +120,9 @@ If you have written a PennyLane-compatible plugin and would like it to be listed
 Developing a plugin
 -------------------
 
-To write your own PennyLane-compatible plugin, the best place to start is our :ref:`overview of the developer API <developer_overview>`, as well as exploring
-the source code of the provided reference plugin modules :mod:`pennylane.plugins.default_qubit` and :mod:`pennylane.plugins.default_gaussian`
+To write your own PennyLane-compatible plugin, the best place to start is our
+:ref:`overview of the plugin developer API <plugin_overview>`, as well as exploring
+the source code of the provided reference plugin modules
+:mod:`pennylane.plugins.default_qubit` and :mod:`pennylane.plugins.default_gaussian`
 (providing the ``default.qubit`` and ``default.gaussian`` devices respectively).
```

Comment:
How about `(these plugins provide the ``default.qubit`` and ``default.gaussian`` devices respectively)`  ?

Reply:
```suggestion
(these plugins provide the ``default.qubit`` and ``default.gaussian`` devices respectively).
```